components/invitation_modal/invitation_modal_members_step/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {connect} from 'react-redux';
+
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getMyTeamMember} from 'mattermost-redux/selectors/entities/teams';
+
+import {GlobalState} from 'types/store';
+
+import {isAdmin} from 'utils/utils.jsx';
+
+import InvitationModalMembersStep from './invitation_modal_members_step';
+
+type OwnProps = {
+    currentTeamId: string;
+}
+
+function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
+    return {
+        userLimit: getConfig(state).ExperimentalCloudUserLimit,
+        currentUsers: state.entities.admin.analytics!.TOTAL_USERS,
+        userIsAdmin: isAdmin(getMyTeamMember(state, ownProps.currentTeamId).roles),
```

Comment:
Should we use here [`getCurrentUser`](https://github.com/mattermost/mattermost-redux/blob/4e60437ef540f030a5fdb9e921451f80f2f2c1ef/src/selectors/entities/common.ts#L42) to use for the `isAdmin` check? It seems more clear on what we are doing that `getMyTeamMember`